code/datums/status_effects/neutral.dm

```diff
@@ -342,3 +342,181 @@
+	alert_type = null
+	processing_flags = STATUS_EFFECT_NORMAL_PROCESS
+	///So we know what cycle we're in during the status
+	var/current_cycle = -250 //Consider it your stability
+	///The addiction looper for addiction stage 3
+	var/phase_3_cycle = -0 //start off delayed
+	///Your clone from another reality
+	var/mob/living/carbon/alt_clone = null
+	///If we display the stabilised message or not
+	var/stable_message = FALSE
+
+/datum/status_effect/eigenstasium/Destroy()
+	QDEL_NULL(alt_clone)
+	. = ..()
+
+/datum/status_effect/eigenstasium/tick()
+	. = ..()
+	//This stuff runs every cycle
+	if(prob(5))
+		do_sparks(5, FALSE, owner)
+
+	//If we have a reagent that blocks the effects
+	var/block_effects = FALSE
+	if(owner.has_reagent(/datum/reagent/bluespace))
+		current_cycle = max(-250, (current_cycle - 7.5)) //cap to -250
+		block_effects = TRUE
+	if(owner.has_reagent(/datum/reagent/stabilizing_agent))
+		current_cycle = max(-250, (current_cycle - 5))
+		block_effects = TRUE
+	var/datum/reagent/eigen = owner.has_reagent(/datum/reagent/eigenstate)
+	if(eigen)
+		if(!eigen.overdosed)
+			current_cycle = max(-250, (current_cycle - 10))
+			block_effects = TRUE
+		else
+			block_effects = FALSE
+
+	if(alt_clone)//catch any stragilers
+		do_sparks(5,FALSE,alt_clone)
+		qdel(alt_clone)
+		owner.visible_message("[owner] is snapped across to a different alternative reality!")
+		alt_clone = null
+
+	if(block_effects)
+		if(!stable_message)
+			owner.visible_message("You feel stable... For now.")
+			stable_message = TRUE
+		return
+	stable_message = FALSE
+
+	//These run on specific cycles
+	switch(current_cycle)
+		if(0)
+			to_chat(owner, "<span class='userdanger'>You feel like you're being pulled across to somewhere else. You feel empty inside.</span>")
+
+		//phase 1
+		if(1 to 50)
+			owner.Jitter(2)
+			owner.adjust_nutrition(-4)
+
+		//phase 2
+		if(50 to 80)
+			if(current_cycle == 51)
+				to_chat(owner, "<span class='userdanger'>You start to convlse violently as you feel your consciousness merges across realities, your possessions flying wildy off your body!</span>")
+				owner.Jitter(200)
+				owner.Knockdown(10)
+			var/items = owner.get_contents()
+			if(!LAZYLEN(items))
+				return ..()
+			var/obj/item/item = pick(items)
+			owner.dropItemToGround(item, TRUE)
+			do_sparks(5,FALSE,item)
+			do_teleport(item, get_turf(item), 3, no_effects=TRUE);
+			do_sparks(5,FALSE,item)
+
+		//phase 3 - little break to get your items
+		if(100 to 150)
+			//Clone function - spawns a clone then deletes it - simulates multiple copies of the player teleporting in
+			switch(phase_3_cycle) //Loops 0 -> 1 -> 2 -> 1 -> 2 -> 1 ...ect.
+				if(0)
+					owner.Jitter(100)
+					to_chat(owner, "<span class='userdanger'>Your eigenstate starts to rip apart, drawing in alternative reality versions of yourself!</span>")
+				if(1)
+					var/typepath = owner.type
+					alt_clone = new typepath(owner.loc)
+					alt_clone.appearance = owner.appearance
```

Comment:
register to the `alt_clone`'s destruction signal here and handle the case of it being destroyed early to avoid hard deletes

Reply:
I think I did what you mean.